test/test_cross_sections.py

```diff
@@ -33,43 +33,47 @@ class TestQuadHex:
     """
 
     def test_constant_lat_cross_section_grid(self):
```

Comment:
Please contains all the testcase for the intersection from the #785 , they are all important and sensitive enough to reflect the problems

Reply:
Let me know if there any other tests needed. I've migrated the ones that don't use any zonal average code (those will stay and be updated in #785)

Reply:
looks good to me